src/routes/components/multi-combobox/+page.svelte

```
@@ -0,0 +1,88 @@
+  />
+  <p class="state-display">Selected: {selectedMediums.join(', ') || '—'}</p>
+</div>
+
+<!-- With a custom action button -->
+<h3>With a custom action button</h3>
+<div class="demo-row" style="max-width: 520px; flex-direction: column; align-items: stretch; gap: 8px;">
+  <MultiCombobox
+    items={campaigns}
+    bind:selected={selectedCampaigns}
+    placeholder="Search campaigns…"
+    onCreate={(v) => { if (!campaigns.some((c) => c.value === v)) campaigns = [...campaigns, { value: v, label: v }]; }}
```

ESLint

[error] Expected { after 'if' condition.

(curly)